lib/locale.t

```diff
@@ -36,7 +36,9 @@ my $is_ebcdic = ord("A") == 193;
 my $os = lc $^O;
 
 # Configure now lets you build a perl that silently ignores taint features
-my $NoTaintSupport = exists($Config{taint_support}) && !$Config{taint_support};
+my $NoTaintSupport =
+    (exists($Config{taint_support}) && !$Config{taint_support}) ||
```

Comment:
I would lose these checks entirely.  I am very hopeful that we will never have a config variable called "taint_support".